hs_core/hydroshare/hs_bagit.py

```diff
@@ -106,10 +104,16 @@ def create_bag_files(resource):
 
     #Set properties of the aggregation
     a._dc.title = resource.title
-    a._dcterms.type = resource._meta.object_name
+    a._dcterms.type = URIRef(resource.metadata.type.url)
     a._citoterms.isDocumentedBy = metadata_url
     a._ore.isDescribedBy = res_map_url
 
+    res_type_aggregation = AggregatedResource(resource.metadata.type.url)
+    res_type_aggregation._rdfs.label = resource._meta.verbose_name
+    res_type_aggregation._rdfs.isDefinedBy = current_site_url + "/terms"
```

Comment:
@pkdash @selimnairb I think @selimnairb's bag ingestion code retrieves resource type class from resourcemap.xml, so it appears this change would break the bag ingestion code. If it is true, we may just have to merge this PR in after the manual/customized migration process although it would be very beneficial if we could merge this PR before the final migration to avoid having to write another script to regenerate xml files for all existing resources. @selimnairb I know migration process has been dragging on much longer than we want and you most likely don't have time to accommodate this additional change for resource type encoding in the xml file. I guess we just move on with migration without incorporating this PR and deal with changes required by merging in this PR later.